lib/rails/generators/anycable/setup/local/local_generator.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+module AnyCableRailsGenerators
+  module Setup
+    # Generator to help set up AnyCable locally
+    class LocalGenerator < ::Rails::Generators::Base
+      SERVER_VERSION = "v0.6.4"
+      OS_NAMES = %w[linux darwin freebsd win].freeze
+      CPU_NAMES = %w[amd64 arm64 386 arm].freeze
+
+      namespace "anycable:setup:local"
+      source_root File.expand_path("templates", __dir__)
+
+      def server
+        answer = nil
+
+        until [1, 2, 3].include?(answer.to_i)
+          answer = ask "How do you want to install AnyCable-Go WebSocket server? (1) Skip, (2) Homebrew, (3) Download binary"
+        end
+
+        case answer.to_i
+        when 1
+          say_status :help, "🛈 You can read about how to install AnyCable WebSocket here 👉 https://docs.anycable.io/#/anycable-go/getting_started", :yellow
+          return
+        when 2
+          run "brew install anycable-go", abort_on_failure: true
+          run "anycable-go -v", abort_on_failure: true
+        when 3
+          return unless download_binary
+        else
+          raise ArgumentError, "Unknown answer: #{answer}"
+        end
+
+        say_status :info, "✔ AnyCable-Go WebSocket server has been installed"
+      end
+
+      def configs
```

Comment:
Let's add `config/anycable.yml` as well.

We already had it in the previous version: https://github.com/anycable/anycable-rails/blob/v0.5.5/lib/generators/anycable/templates/anycable.yml (it contains a lot of useful annotations).
It's a bit outdated (e.g., it doesn't contain `access_logs_disabled: false` parameter).